nym-vpn-apple/Services/Sources/Services/AppSettings/AppSettings.swift

```diff
@@ -78,4 +78,6 @@ enum AppSettingKey: String {
     case connectionType
     case lastConnectionIntent
     case currentEnv
+    case didConnectSuccessfullyOnce
```

Comment:
I don't think that neither `didConnectSuccessfullyOnce` nor `didAskForNotificationPermission` are used.